src/main/java/com/ldtteam/blockui/util/SafeError.java

```diff
@@ -0,0 +1,27 @@
+package com.ldtteam.blockui.util;
+
+import com.ldtteam.blockui.mod.Log;
+import net.minecraftforge.fml.loading.FMLEnvironment;
+
+/**
+ * Utility class for throwing errors which is safe during production.
+ */
+public class SafeError
+{
+    /**
+     * Safe error throw call that only throws an exception during development, but logs an error in production instead so no crashes to desktop may occur.
+     *
+     * @param exception the exception instance.
+     */
+    public static <T extends Exception> void throwOrLog(final T exception) throws T
```

Comment:
maybe throwInProd?

Reply:
No we don't want to throw in production lol

Reply:
But thats what it does? Isnt it?

Reply:
If we're in production it logs a message, if it's in development it throws an exception and will crash. So it's not "throw in production", it's "throw in development"

Reply:
Sorry, I mean call it throwInDev